src/other/AutoCompleteText.js

```diff
@@ -15,6 +15,7 @@
     AutoCompleteText.prototype._class += " other_AutoCompleteText";
 
     AutoCompleteText.prototype.publish("label", "Label: ", "string", "Label for AutoCompleteText");
+    AutoCompleteText.prototype.publish("textboxWidth", 150, "number", "width of textbox");
```

Comment:
Question - do you ever want it to be just a a "default size"?  You could do it like this:
```
AutoCompleteText.prototype.publish("textboxWidth", null, "number", "width of textbox", null, {optional:true});
...
this._input.style("width",this.textboxWidth_exists() ? this.textboxWidth()+"px" : null);
```